canonical-web-and-design/snapcraft.io
# Update kde snaps (as icons have updated etc.)

## QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/publisher/kde
- Make sure all the snaps click through and look ok
